pkg/che/che.go

```diff
@@ -204,3 +205,39 @@ func readCheUserFromJSON(jsonString string) (*User, error) {
 	}
 	return &cheUser, nil
 }
+
+// DevSpacesDBCleanerDelete deletes the user from the Dev Spaces database via the che-db-cleaner service
+// curl -X DELETE http://che-db-cleaner.crw/<userid>
+func (c *Client) DevSpacesDBCleanerDelete(userID string) error {
+
+	config, err := membercfg.GetConfiguration(c.k8sClient)
+	if err != nil {
+		return err
+	}
+
+	// Dev Spaces URL
+	devSpacesURL := fmt.Sprintf("http://che-db-cleaner.%s/%s", config.Che().Namespace(), userID)
+
+	// create request
+	req, err := http.NewRequestWithContext(context.TODO(), http.MethodDelete, devSpacesURL, nil)
+	if err != nil {
+		return errors.Wrapf(err, "unable to create Dev Spaces delete request")
+	}
+
+	// do the request
+	res, err := c.httpClient.Do(req)
+	if err != nil {
+		return errors.Wrapf(err, "unable to delete Dev Spaces user with ID '%s'", userID)
+	}
+	defer rest.CloseResponse(res)
+	if res.StatusCode != http.StatusOK && res.StatusCode != http.StatusNotFound {
+		resBody, readError := rest.ReadBody(res.Body)
+		if readError != nil {
+			log.Error(readError, "error while reading body of the delete Dev Spaces user response")
+		}
+		err = errors.Errorf("unable to delete Dev Spaces user with ID '%s', Response status: '%s' Body: '%s'", userID, res.Status, resBody)
+	} else if res.StatusCode == http.StatusNotFound {
```

Comment:
you could probably switch the `if`/`else if` statements to simplify the code, or even use a `switch` statement

Reply:
I'm not sure how switching would simplify it. Do you have an example?

Reply:
something like this:
```
switch res.StatusCode {
case http.StatusOK
  ...
case http.StatusNotFound
  ...
default:
  ...
}
```

Reply:
that being said, it's just a suggestion. Feel free to keep it as-is if you think it's better ;)

Reply:
Actually, you're right the switch statement is more readable. Done in 0e60c78ad3da40600118153d2f535388f1d3dfec Thanks for the suggestion!